smoke-tests/apps/RuntimeAttachWithDelayedConnectionString/src/main/resources/applicationinsights.json

```diff
@@ -0,0 +1,10 @@
+{
+  "startWithoutConnectionString": true,
```

Comment:
When the user uses the runtime attachment API, I feel it would be more natural and convenient to enable the "start without connection string" mode in a programmatic way.

Reply:
planning to wait on adding API for now, will see user feedback / amount of usage